index.html

```diff
@@ -39,6 +39,18 @@ <h1 class="CUserInterface">
 			<button id="addBtn" onclick="addToJSON()">Add post!</button>
 		</center>		
 
+		<div class="searchBar">
+			<div class="tags">
```

Comment:
Is there a reason why these elements are in two nested <div> tags? Could they be in just one tag?